### PR TITLE
Changed ReactiveCocoa branch name

### PR DESCRIPTION
**Pull request by AndrewSB**
ReactiveCocoa no longer has the swift-development branch
Moving to the swift-1.2 branch

### CODE REVIEW

**Comment by ikesyo** [comment]
Thank you for the pull request! However, that `swift-1.2` branch is outdated (I don't know why the branch still exists). So we should use the latest `v3.0-RC.1` release tag instead.

**Comment by AndrewSB** [comment]
thanks @ikesyo made the change!

File: Cartfile [hunk 1]
```diff
@@ -1 +1 @@
-github "ReactiveCocoa/ReactiveCocoa" "swift-development"
+github "ReactiveCocoa/ReactiveCocoa" "6c37b5ef416799a3a2a61ff78f051d5deb0d76b2"
```
Comment (ikesyo) [inline Cartfile]: Could you please use `v3.0-RC.1` tag, not the commit SHA?

Comment (ikesyo) [inline Cartfile]: Additionally, `Cartfile.resolved` and the submodule reference should be updated too. You could use `$ carthage update --use-submodules --no-build --no-use-binaries`.

**Comment by AndrewSB** [comment]
Sorry about that @ikesyo! Didn't realize `git rev-parse` could reference tags as well

**Comment by ikesyo** [comment]
Looks good! :+1: Thanks for your work. :sparkling_heart:

**Pull request merged**